src/client/Microsoft.Identity.Client/Platforms/Android/AndroidPlatformProxy.cs

```diff
@@ -233,7 +233,16 @@ public override IBroker CreateBroker(CoreUIParent uIParent)
 
         public override bool CanBrokerSupportSilentAuth()
         {
-            return true;
+            IBroker broker = CreateBroker(null);
```

Comment:
i like this better. but would like to get BG's thoughts before we merge.

Reply:
Me too.